### ISSUE TITLE
Re-enable "aria-allowed-attr" in a11y check

### ISSUE THREAD

**Issue by mcoker** [patternfly/patternfly#4201]
When we [built the tree table](https://github.com/patternfly/patternfly/pull/3846), we had to disable the `aria-allowed-attr` check in our a11y test config.

https://github.com/patternfly/patternfly/blob/0853dbc02fa5bdc83ac57cec14a55f13518cd50d/patternfly-a11y.config.js#L12

I'd like to see if there is a way that we can re-enable that that check for `aria-allowed-attr`, it has helped us identify valid a11y issues in the past. 

I'm no longer seeing axe errors on the [w3 treegrid examples](https://www.w3.org/TR/wai-aria-practices/examples/treegrid/treegrid-1.html), however I am still seeing the errors when running the a11y check locally.

I'm wondering if there is a newer version of the axe tool that we use for our a11y check? If not, is there another way we could re-enable this check?

FWIW, these are the example pages that currently fail that check:

```
515/818   /components/table/html/tree-table-with-checkboxes
violations: aria-allowed-attr

516/818   /components/table/html/tree-table
violations: aria-allowed-attr

519/818   /components/table/html/tree-table-with-checkboxes-icons
violations: aria-allowed-attr
```

cc @redallen @jessiehuff
